EIPS/eip-5023.md

```diff
@@ -0,0 +1,145 @@
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+```solidity
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+
+interface IERCsntNFT is IERC165 {
+
+  /// @dev This emits when a token is shared, reminted and given to another wallet that isn't function caller
+  event Share(address indexed from, address indexed to, uint256 indexed tokenId, uint256 derivedFromtokenId);
+
+  /// @dev Shares, remints an existing token, gives a newly minted token a fresh token id, keeps original token at function callers possession and transfers newly minted token to receiver which should be another address than function caller. 
+  function share(address to, uint256 tokenIdToBeShared, uint256 newTokenId) external virtual;
+
+} 
+```
+
+## Rationale
+
+Current NFT standards define transferable non-fungible tokens. However to be able to create shareable NFTs we see that existing NFT contracts could be extended with an interface which defines the basic principles of sharing, namely the Event of sharing and the function method of sharing. In ATARCA we have chosen to go with shareable non-transferable NFTs which in our use case denote an award or merit for done effort. 
+
+Shareable tokens can be transferable or non-transferable as it is up to implementation details of the token contract. In the reference implementation we have a general case distilled from the ATARCA case that defines a shareable non-transferable NFTs using the shareable NFT interface.
+
+
+
+## Backwards Compatibility
+
+TBD
+
+## Reference Implementation
+
+Following reference implementation demonstrates a general use case of one of our pilots. In this case a shareable non-transferable token represents a contribution done to a community that the contract owner has decided to merit with a token. Contract owner can mint a merit token and give it to a person. This token can be further shared by the receiver to other parties for example to share the received merit to others that have participated or influenced his contribution.
+
+```
+// SPDX-License-Identifier: MIT
```

Comment:
Should be CC0 licensed (as the whole EIP is CC0).

Reply:
Good catch! Will incorporate the updated licence in next commit.